org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/importing/CsarImporter.java

```diff
@@ -434,6 +434,9 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
                 this.adjustServiceTemplate(defsPath.getParent().getParent(), tmf, (ServiceTemplateId) wid, (TServiceTemplate) ci, errors);
                 entryServiceTemplate = Optional.of((ServiceTemplateId) wid);
             }
+            
+            // import license and readme files
```

Comment:
No need to add this comment.See [Java-by-Comparison](http://java.by-comparison.com/) issue "Remove Superfluous Comments"

Reply:
sure!
It seems that there's no standalone CsarImporterTest, @koppor do you want to have a test for Csar import in CsarExporterTest or in a dedicated CsarImporterTest?

Reply:
Yeah, good idea for having a CsarImporterTest. You can copy and paste code from CsarExporterTest (the thing with `extends TestWithGitBackedRepository {`) 😇 

Oh, wow, generalization, so that it imports/exports any file there not generated by Winery (*.mimetype, *.tosca). No recursion into subdirectories. -- If that is easy to implement.